<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogArchivi</class>
 <widget class="QDialog" name="DialogArchivi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#DialogArchivi{
background-color:qlineargradient(spread:pad, x1:0.058, y1:0.136364, x2:1, y2:1, stop:0 rgba(0, 255, 127, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>801</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;color:rgb(255, 255, 255);
font-weight:bold;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;SEZIONE ARCHIVI&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="archvenditeBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
border :2px solid ;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;
background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string> VENDITE</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/cashh.png</normaloff>img/cashh.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="archclientiBtn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>360</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
border :2px solid ;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;
background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>CLIENTI</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/clie.png</normaloff>img/clie.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="archordiniButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>590</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
border :2px solid ;
font: 14pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;
background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>ORDINI</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/del.png</normaloff>img/del.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="archesitiBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>590</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
border :2px solid ;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;
background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string> ESITI</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/te.png</normaloff>img/te.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>90</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="homeBtn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>510</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/home.png</normaloff>img/home.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1271</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/farmaciaa.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>archvenditeBtn</zorder>
  <zorder>archclientiBtn</zorder>
  <zorder>archordiniButton</zorder>
  <zorder>archesitiBtn</zorder>
  <zorder>homeBtn</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
